action.yml

```diff
@@ -38,6 +38,9 @@ inputs:
   allow_reresolve:
     description: 'Whether to allow re-resolving of package versions in the test environment. Only effective on Julia 1.9+. Options: true | false. Default value: true'
     default: 'true'
+  test_arg:
+    description: 'Argument string that is passed on to test.'
+    default: ''
```

Comment:
Just thinking this through given we want to add multiple args after this PR.. should we just call this `test_args` for this PR, and wait to release until after its updated to a list in the following PR?